tests/test_project.py

```diff
@@ -98,6 +98,11 @@ def test_workspace_directory_with_env_variable(self):
         self.project.config['workspace_dir'] = '${SIGNAC_ENV_DIR_TEST}'
         self.assertEqual(self._tmp_wd, self.project.workspace())
 
+    def test_workspace_directory_exists(self):
+        root = self._tmp_dir.name
```

Comment:
There is already a project that is initialized by the setup for all the tests in this section. The class `ProjectTest` inherits from `BaseProjectTest`, which in turn inherits from `BaseJobTest` in `test_job.py`. The method `setUp` in that class initializes a project and makes it accessible via `self.project`. https://github.com/glotzerlab/signac/blob/dc870ac06159cae97e1475894913d1def567555e/tests/test_job.py#L71-L81

By using `self.project`, the contents of this test function can be simplified to just one line that uses the existing project:
```python
def test_workspace_directory_exists(self):
    self.assertTrue(os.path.exists(self.project.workspace()))
```